src/content/docs/en/reference/experimental-flags/csp.mdx

```diff
@@ -21,7 +21,7 @@ Enables support for [Content Security Policy (CSP)](https://developer.mozilla.or
 
 Enabling this feature adds additional security to **Astro's handling of processed and bundled scripts and styles** by default, and allows you to further configure these, and additional, content types.
 
-This experimental CSP feature has some limitations. Inline scripts are not supported out of the box, but you can [provide your own hashes](#hashes) for external and inline scripts. Additionally, [Astro's view transitions](/en/guides/view-transitions/) using the `<ClientRouter />` are not yet fully supported: when navigating from one page to another, some styles may not be applied and some scripts may not be executed.
+This experimental CSP feature has some limitations. Inline scripts are not supported out of the box, but you can [provide your own hashes](#hashes) for external and inline scripts. Since v5.9.2, [Astro's view transitions](/en/guides/view-transitions/) using the `<ClientRouter />` are not supported anymore.
```

Comment:
I'd just say it's not supported. We don't need to preserve the fact that it was supposedly supported but didn't work for a few days